dev/galoy-deps/Tiltfile

```diff
@@ -0,0 +1,66 @@
+load('ext://helm_resource', 'helm_resource')
+load('ext://namespace', 'namespace_create')
+load('../common/Tiltfile', 'copy_secret', 'helm_release')
+update_settings(k8s_upsert_timeout_secs=120)
+
+name_prefix = "galoy-dev"
+kafka_namespace = "{}-kafka".format(name_prefix)
+ingress_namespace = "{}-ingress".format(name_prefix)
+otel_namespace = "{}-otel".format(name_prefix)
+kubemonkey_namespace = "{}-kubemonkey".format(name_prefix)
+
+namespace_create(kafka_namespace)
+
+# Attaching the namespace to the workload means that the namespace will be deleted when the workload is deleted
+# k8s_resource(workload="kafka-operator", objects=["galoy-dev-kafka:namespace"])
+
+#helm_release( '../../charts/galoy-deps',
+#  name='kafka',
+#  namespace=kafka_namespace,
+#  values=['./kafka-values.yml'],
+#  dependency_build=True,
+#  add_repos=True
+#)
+
+## cert-manager and ingress-nginx
+
+namespace_create(ingress_namespace)
+
+# TODO: decide how to label the ingress namespace
+# k8s_resource(workload="cert-manager?", objects=["galoy-dev-ingress:namespace"])
+
+#helm_release(
+#  '../../charts/galoy-deps',
+#  name='cert-manager',
+#  namespace=ingress_namespace,
+#  values=['./cert-manager-values.yml'],
+#)
+
+#helm_release(
+#  '../../charts/galoy-deps',
+#  name='ingress-nginx',
+#  namespace=ingress_namespace,
+#  values=['./ingress-nginx-values.yml'],
+#)
+
+## opentelemetry-collector
+
+namespace_create(otel_namespace)
+
+#helm_release(
+#  '../../charts/galoy-deps',
+#  name='opentelemetry-collector',
+#  namespace=otel_namespace,
+#  values=['./otel-values.yml'],
+#)
+
+# do we need kubemonkey in local dev?
+
+# helm_resource(
+#   name="kubemonkey",
+#   chart="../../charts/galoy-deps",
+#   namespace=kubemonkey_namespace,
+#   flags=['--values=./kubemonkey-values.yml'],
+#   labels="kubemonkey"
+# )
```

Comment:
need to have all these enabled

Reply:
@krtk6160 We don't have galoy-deps chart available, should we copy them over?

Reply:
Yes